migrations/V19__Add_Hosting_And_UI_To_Business_Case.sql

```diff
@@ -0,0 +1,15 @@
+ALTER TABLE business_case ADD COLUMN preferred_hosting_type text;
+ALTER TABLE business_case ADD COLUMN preferred_hosting_location text;
+ALTER TABLE business_case ADD COLUMN preferred_hosting_cloud_service_type text;
+ALTER TABLE business_case ADD COLUMN preferred_has_ui text;
```

Comment:
This can be yes, no, or not sure. That's why it's not a boolean, but I'm also unsure if it should be a enum of some sort.